docs/reference/kubectl/jsonpath.md

```diff
@@ -8,6 +8,8 @@ And we add three functions in addition to the original JSONPath syntax:
 1. The `$` operator is optional since the expression always starts from the root object by default.
 2. We can use `""` to quote text inside JSONPath expressions.
 3. We can use `range` operator to iterate lists.
+4. We can use negative slice indices to step backwards through a list.
+  - Negative indices do not "wrap around" a list. They are valid as long as `-index + listLenght >= 0`.
```

Comment:
Is it possible to combine this bullet with the 4th item into a single line? I personally feel a bit strange to see a single bullet here. But it's also fine to me as-is anyway :)

Reply:
Updated to a single line, thanks